### ISSUE TITLE
Баг с капюшоном зимнего костюма

### ISSUE THREAD

**Issue by Mintt69** [TauCetiStation/TauCetiClassic#2929]
#### Подробное описание проблемы
Вобщем, когда надеваешь капюшон костюма красного, который добавили в шкафчики СБ, и у тебя уже надет головной убор, то при снятии как капюшона так и головного убора остается иллюзия будто головной убор надет, так можно стакать головные уборы+ когда у меня была прическа лысо-серая, я много раз надел-снял капюшон и прическа стала полностью черная

**Comment by RichardJones1** [comment]
Да, дубликат недавнего ишью, но в этом также есть про прически.
Я смогу пофиксить баг с шапками, но с прической - не могу... Не получается.

Вижу несколько вариантов - убрать эти куртки вообще, пофиксить баг, либо взять их код с тг. Спойлер - там они совсем другие.

Спасибо за то что написал ишью.

**Comment by RichardJones1** [comment]
Nevermind, у меня таки получилось XD
